e2e/testcases/correction-birth/correct-birth-record-4.spec.ts

```diff
@@ -231,17 +238,14 @@ test.describe.serial(' Correct record - 4', () => {
 
         await expect(
           page.locator('#father-content #Date').getByRole('deletion')
-        ).toHaveText(
-          format(parseISO(declaration.father.birthDate), 'dd MMMM yyyy'),
-          { ignoreCase: true }
-        )
+        ).toHaveText(formatDateTo_ddMMMMyyyy(declaration.father.birthDate), {
```

Comment:
Is the case something we need to ignore? Would be nice if the formatting was all we needed